todo-list-service/src/main/java/org/example/todo_list_service/TodoListServiceApplication.java

```diff
@@ -0,0 +1,36 @@
+package org.example.todo_list_service;
+
+import java.util.ArrayList;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@SpringBootApplication
+public class TodoListServiceApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(TodoListServiceApplication.class, args);
+	}
+
+	@RestController
+	public class HelloController {
+
+		@GetMapping("/hello")
+		public String sayHello(String name, String lastName) {
+			return "¡Hello World!";
+		}
+
+		@GetMapping("/todos")
+		public ArrayList<Todos> getMockedTodos() {
```

Comment:
Update the signature like this:

`public List<Todos> getAllTodos()`

* Changing `ArrayList` to `List`: Depending on base classes rather than derived/concretes enables us to use derived classes transparently (for example: this will work even if the implementation is done with ArrayList, LinkedList, or other List's subclasses.) . This is also a SOLID principle: https://en.wikipedia.org/wiki/SOLID#Liskov_substitution_principle
* Don't need to name the method `getMockedTodos` even though the data is currently hard-coded. We can already use the method final name and treat it as placeholder using the mocked data.